src/pages/proposal.tsx

```diff
@@ -102,13 +101,11 @@ export default function ProposalPage() {
   const [voting, vote] = useAsyncTask(
     async (_: string, choiceIndex: number) => {
       if (proposal && account && provider && votes) {
-        const message = await Snapshot.get().createVoteMessage(
-          proposal.snapshot_space,
-          proposal.snapshot_id,
-          choiceIndex
+        const web3Provider = new Web3Provider(provider)
+        const [listedAccount] = await web3Provider.listAccounts()
```

Comment:
Why can't we use `const [account] = useAuthContext()` here?

Reply:
Good point.
`listAccounts()` returns a checksumed address while `useAuthContext()` doesn't, so it wouldn't work.
And in snapshot docs they use it like this, so it seemed right.
We could checksum addresses ourselves, but I didn't think it was necessary just for this case.